backend/src/poap/index.js

```diff
@@ -3,7 +3,7 @@ import axios from 'axios';
 class POAP {
 
     static async fetchEvents() {
-        const response = await axios.get('https://api.poap.xyz/events');
+        const response = await axios.get('https://api-event.poap.xyz');
```

Comment:
Create a Jira ticket with technical debt to move this URL to an environment variable and add the ticket in this comment.